### PR TITLE
Update OWNERS_ALIASES

### PR DESCRIPTION
**Pull request by Rajakavitha1**
Added rajakavitha1 to the approver list.

>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> Please delete this note before submitting the pull request.
>
> For 1.14 Features: set Milestone to 1.14 and Base Branch to dev-1.14
> 
> For Chinese localization, base branch to release-1.12
>
> For Korean Localization: set Base Branch to dev-1.13-ko.<latest team milestone>
>
> Help editing and submitting pull requests:
> https://kubernetes.io/docs/contribute/start/#improve-existing-content.
>
> Help choosing which branch to use:
> https://kubernetes.io/docs/contribute/start#choose-which-git-branch-to-use.
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>

### CODE REVIEW

File: OWNERS_ALIASES [hunk 1]
```diff
@@ -122,14 +122,15 @@ aliases:
...
+    - rajakavitha1
     - ryanmcginnis
     - steveperry-53
     - stewart-yu
     - tengqm
     - tfogo
     - zacharysarah
     - zhangxiaoyu-zidif
-    - zparnold
+    - zparnold
```
Comment (zacharysarah) [inline OWNERS_ALIASES]: Please configure your editor not to add or remove trailing whitespace.

Comment (Rajakavitha1) [reply OWNERS_ALIASES]: okay! thank you :)

**Comment by Rajakavitha1** [comment]
Changed the settings in the editor. Opening a new PR.

**Pull request closed without merging**